tests/specs/library.test.js

```diff
@@ -0,0 +1,34 @@
+import child_process from 'node:child_process';
+import fs from 'node:fs';
+import path from 'node:path';
+import process from 'node:process';
+
+import { describe, expect, test } from 'vitest';
+
+import { run } from '../../library.js';
+
+describe('Library', () => {
+  describe('Run', () => {
+    test('Handles versions with hyphenated build numbers', async () => {
```

Comment:
The pattern I use in my tests is to have one test for each file (so I renamed this file). In the test file, I'll have the entire test wrapped in a `describe` with the file name (helpful when running `npm t` locally to know what failed), then for the case of libraries have a `describe` for the function I am testing, then inside that have tests for different scenarios that function needs to handle with names explaining the reason the test exists.